lib/bigtable/mutation.js

```diff
@@ -0,0 +1,320 @@
+/*!
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ /*!
+ * @module bigtable/mutation
+ */
+
+'use strict';
+
+var is = require('is');
+var arrify = require('arrify');
+
+/**
+ * Formats table mutations to be in the expected proto format.
+ *
+ * @constructor
+ * @alias module:bigtable/mutation
+ *
+ * @example
+ * var mutation = new Mutation({
+ *   key: 'gwashington',
+ *   method: 'insert',
+ *   data: {
+ *     jadams: '1'
+ *   }
+ * });
+ */
+function Mutation(mutation) {
```

Comment:
Do we have an Insert and Update, or just Mutation?  Seems less friendly than it could be.